src/client/waiter.ts

```diff
@@ -64,7 +87,7 @@ export class Waiter {
 
   log(s: string) {
     this._logs.push(s);
-    debugLogger.log('api', s);
+    this._channelOwner._waitForEventInfoLog(this._waitId, s);
```

Comment:
This makes me think whether we can now remove api call logging from the client and emit it from the server? We should have all the information already.

Reply:
Yes, that's the goal. We need to add api call name to metadata for that.